main.js

```diff
@@ -0,0 +1,90 @@
+
+const inputCountry = document.getElementById("inputCountry");
+ inputCountry.addEventListener('keypress', (e) => {
+     if (e.key === "enter") {
```

Comment:
Dit if statement zal nooit true zijn. Dat komt omdat het hoofdlettergevoelig is. Hier had `if (e.key === "Enter")` moeten staan